sql-cli/examples/_dags/simple.py

```diff
@@ -0,0 +1,82 @@
+# NOTE: This is an auto-generated file. Please do not edit this file manually.
+from airflow import DAG
+from airflow.utils import timezone
+from astro import sql as aql
+from astro.sql.table import Table
+
+with DAG(
+    dag_id="simple",
+    start_date=timezone.parse("2022-09-20"),
+) as dag:
+    a = aql.transform_file(
```

Comment:
Assuming this is based on the file name, we should have better file names so that our variables have more interesting names. Single letter variables in our example DAGs won't give an impression of production readiness.

Reply:
I agree with you. This was only for testing purpose. I need to come up with some more realistic names. If you have any suggestion, please let me know :)

Reply:
fixed in c6902aa